### DISCUSSION TITLE
CodeMaid is missing in context menu of json editor (VS 2022 17.2.6)

### DISCUSSION THREAD

**Discussion by slava-sakidon** [codecadwallader/codemaid#935] [Q&A] [answered]
Unfortunately I don't see the Code Maid commands in json editor of VS 2022. When this problem will be resolved? Thanks
![image](https://user-images.githubusercontent.com/111275633/184606861-ba7c9d05-c38d-4c11-bbf6-82da56e31c79.png)

**Comment by codecadwallader** [comment, marked as answer]
Thanks for reporting it, I have moved this to an issue #939
